Demos/iosAppSwift/iosAppSwift/iosAppSwiftApp.swift

```diff
@@ -22,7 +22,7 @@ class AppDelegate: NSObject, UIApplicationDelegate {
     ) -> Bool {
         // Dynamically read these settings from your config settings on
         // application startup.
-        let accessToken = "YOUR-ROLLBAR-ACCESSTOKEN"  // Rollbar post_client_item access token
+        let accessToken = "2fa95b91f38448299b5097ff11cffb23"  // Rollbar post_client_item access token
```

Comment:
@matux we need to revoke this token and remove it, right?

Reply:
Not sure about revoking, I'll check the account I'm using. I just removed it from the code, tho.